cmd/installer/cli/cidr.go

```diff
@@ -0,0 +1,44 @@
+package cli
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/netutils"
+	"github.com/spf13/cobra"
+)
+
+// DeterminePodAndServiceCIDRS determines, based on the command line flags,
+// what are the pod and service CIDRs to be used for the cluster. If both
+// --pod-cidr and --service-cidr have been set, they are used. Otherwise,
+// the cidr flag is split into pod and service CIDRs.
+func determinePodAndServiceCIDRs(cmd *cobra.Command) (string, string, error) {
+	podCIDRFlag, err := cmd.Flags().GetString("pod-cidr")
+	if err != nil {
+		return "", "", fmt.Errorf("unable to get pod-cidr flag: %w", err)
+	}
+	serviceCIDRFlag, err := cmd.Flags().GetString("service-cidr")
+	if err != nil {
+		return "", "", fmt.Errorf("unable to get service-cidr flag: %w", err)
+	}
+
+	if podCIDRFlag != "" && serviceCIDRFlag != "" {
```

Comment:
this needs to be an `||`
```suggestion
	if podCIDRFlag != "" || serviceCIDRFlag != "" {
```